pilot/helpers/Project.py

```diff
@@ -82,9 +82,9 @@ def __init__(
         self.clarifications = None
         self.user_stories = None
         self.user_tasks = None
-        self.architecture = None
-        self.system_dependencies = None
-        self.package_dependencies = None
+        self.architecture = ""
+        self.system_dependencies = []
+        self.package_dependencies = []
```

Comment:
Sane defaults here aren't really needed, but just in case there's a bug with Architect in the future where it doesn't set these. (this was found via a unit test which doesn't go through Architect).